commit 1885b97f2645db94c14f6a5652ea4b34c7e61b77
Author: Stefan Wahren <stefan.wahren@i2se.com>
Date:   Sat Oct 20 20:25:41 2018 +0200

    staging: vchiq_arm: Register a platform device for the audio driver
    
    Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>

diff --git a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
index e62222549523..b8f5d07a1b69 100644
--- a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
+++ b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
@@ -170,6 +170,7 @@ static struct class  *vchiq_class;
 static struct device *vchiq_dev;
 static DEFINE_SPINLOCK(msg_queue_spinlock);
 static struct platform_device *bcm2835_camera;
+static struct platform_device *bcm2835_audio;
 
 static const char *const ioctl_names[] = {
 	"CONNECT",
@@ -3654,6 +3655,7 @@ static int vchiq_probe(struct platform_device *pdev)
 		MAJOR(vchiq_devid), MINOR(vchiq_devid));
 
 	bcm2835_camera = vchiq_register_child(pdev, "bcm2835-camera");
+	bcm2835_audio = vchiq_register_child(pdev, "bcm2835_audio");
 
 	return 0;
 
@@ -3670,6 +3672,8 @@ static int vchiq_probe(struct platform_device *pdev)
 
 static int vchiq_remove(struct platform_device *pdev)
 {
+	if (!IS_ERR(bcm2835_audio))
+		platform_device_unregister(bcm2835_audio);
 	if (!IS_ERR(bcm2835_camera))
 		platform_device_unregister(bcm2835_camera);
 	vchiq_debugfs_deinit();
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1380_1885b97f2645db94c14f6a5652ea4b34c7e61b77.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1380_1885b97f2645db94c14f6a5652ea4b34c7e61b77.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
